style.css

```diff
@@ -492,3 +521,232 @@ form textarea {
   color: #5e6c84;
   border: 1px solid #c1c7d0;
 }
+
+/* Medium devices (landscape tablets, 768px and up) */
+@media only screen and (min-width: 768px) {
+  #menu-icon {
+    display: none;
+  }
+
+  a {
+    text-decoration-line: none;
+    color: #344563;
+  }
```

Comment:
- [ ] Kindly make sure the color of the links on the navigation bar is the same as the one on the provided Figma design.

  - ### YOUR DESIGN
  
     ![image](https://user-images.githubusercontent.com/87197412/222296237-86129687-6ca9-4160-b11c-1b59279d4874.png)

  - ### FIGMA DESIGN
   ![image](https://user-images.githubusercontent.com/87197412/222296282-ecc85232-bfae-4788-97ec-21ccbf9d2cbe.png)